### PR TITLE
Add restart values to the supervisors lesson

### PR DESCRIPTION
**Pull request by brain-geek**
The #658 issue got me an idea about improving the existing lesson.

While it's additional info, I guess that understanding this concept is important.

### CODE REVIEW

File: lessons/advanced/otp-supervisors.md [hunk 1]
```diff
@@ -69,12 +79,14 @@ Including the `Task.Supervisor` is no different than other supervisors:
...
 
 children = [
-  supervisor(Task.Supervisor, [[name: ExampleApp.TaskSupervisor]]),
+  supervisor(Task.Supervisor, [[name: ExampleApp.TaskSupervisor, restart: :transient]]),
 ]
 
 {:ok, pid} = Supervisor.start_link(children, strategy: :one_for_one)
 ```
 
+Please note, that
```
Comment (doomspork) [inline lessons/advanced/otp-supervisors.md]: Did you leave something off here @brain-geek?

File: lessons/advanced/otp-supervisors.md [hunk 2]
```diff
@@ -42,6 +42,16 @@ There are currently four different restart strategies available to supervisors:
 
 + `:simple_one_for_one` - Best for dynamically attached children. Supervisor spec is required to contain only one child, but this child can be spawned multiple times. This strategy is intended to be used when you need to dynamically start and stop supervised children.
 
+### Restart values
+
+There are several approaches for handling child process crashes:
+
+* `:permanent` - Child is always restarted.
```
Comment (ybur-yug) [inline lessons/advanced/otp-supervisors.md]: In the next two lines a different style in the form of `the child` is what you use. Either could be reasonably considered correct. I would stick with one style for all 3. (nitpick, not real review comment)

Comment (nscyclone) [inline lessons/advanced/otp-supervisors.md]: Worth mentioning that the definitions from the strategies list (several lines above) start with a capital letter.

**Comment by brain-geek** [comment]
@doomspork @ybur-yug @nscyclone Thanks for review!

Can you look through the PR again? :)

**Pull request merged**